### PR TITLE
Improve readme for old PHP versions

### PR DESCRIPTION
**Pull request by remi-stripe**
Update readme to make it clearer how to use (or not use) old PHP versions.

r? @richardm-stripe 
cc @stripe/api-libraries

### CODE REVIEW

**Review by richardm-stripe — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -76,11 +76,13 @@ See [video demonstrations][youtube-playlist] covering how to use the library.
 
 ### PHP 5.4 & 5.5
 
-If you are using PHP 5.4 or 5.5, you can download v6.21.1 ([zip](https://github.com/stripe/stripe-php/archive/v6.21.1.zip), [tar.gz](https://github.com/stripe/stripe-php/archive/v6.21.1.tar.gz)) from our [releases page](https://github.com/stripe/stripe-php/releases). This version will continue to work with new versions of the Stripe API for all common uses.
+If you are using PHP 5.4 or 5.5, you should consider upgrading your environment as those versions have been past end of life since September 2015 and July 2016 respectively.
+Otherwise, you can still use Stripe by downloading stripe-php v6.43.1 ([zip](https://github.com/stripe/stripe-php/archive/v6.43.1.zip), [tar.gz](https://github.com/stripe/stripe-php/archive/v6.21.1.tar.gz)) from our [releases page](https://github.com/stripe/stripe-php/releases). This version will work but might not support recent features we added since the version was released and upgrading PHP is the best course of action.
```
Comment (lode) [inline README.md:80]: I don't know if the change from `6.21.1` to `6.43.1` was intentional, but at least the zip and tar.gz links point to different releases.

Comment (remi-stripe) [reply README.md:80]: @lode it was intentional. We didn't drop support for old PHP version until the next major, so the link points to the most recent minor version for 6.x with support for those PHP versions.

Comment (remi-stripe) [reply README.md:80]: ah damn I see what you meant, I only fixed one, sorry. Nice catch. Let me amend quickly in another PR

Comment (remi-stripe) [reply README.md:80]: Okay fixed in https://github.com/stripe/stripe-php/pull/1161

Comment (lode) [reply README.md:80]: Thanks! :sparkles:

**Pull request merged**